packages/blade/src/components/Tabs/tabTokens.ts

```diff
@@ -105,70 +105,68 @@ const paddingX: TabItemPadding = {
   },
 };
 
+const trackColor = 'surface.border.gray.muted';
 const textColor = {
   selected: {
-    default: 'brand.primary.500',
-    hover: 'brand.primary.600',
-    focus: 'brand.primary.600',
-    active: 'brand.primary.600',
-    disabled: 'surface.text.placeholder.lowContrast',
+    default: 'interactive.text.primary.subtle',
+    highlighted: 'interactive.text.primary.normal',
+    disabled: 'interactive.text.primary.normal',
   },
   unselected: {
-    default: 'surface.text.muted.lowContrast',
-    hover: 'surface.text.subdued.lowContrast',
-    focus: 'surface.text.subdued.lowContrast',
-    active: 'surface.text.subdued.lowContrast',
-    disabled: 'surface.text.placeholder.lowContrast',
+    default: 'interactive.text.gray.muted',
+    highlighted: 'interactive.text.gray.subtle',
+    disabled: 'interactive.text.gray.disabled',
   },
 } as const;
 
 const backgroundColor = {
   unselected: {
     bordered: {
-      default: 'transparent',
-      hover: 'transparent',
-      focus: 'transparent',
-      active: 'transparent',
-      disabled: 'transparent',
+      default: 'colors.transparent',
+      highlighted: 'colors.transparent',
+      disabled: 'colors.transparent',
     },
     borderless: {
-      default: 'transparent',
-      hover: 'transparent',
-      focus: 'transparent',
-      active: 'transparent',
-      disabled: 'transparent',
+      default: 'colors.transparent',
```

Comment:
nit: remove this `colors` prefix I guess? looks like we earlier had it because `transparent` token didn't exist

Reply:
We have to keep colors. prefix because now the transparent is not a color it's a token which we need to `getIn(theme, `colors.transparent`)`